src/App.tsx

```diff
@@ -15,37 +18,127 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+// export const App: React.FC = () => {
+//   return (
+//     <div className="section content">
+//       <div className="buttons">
+//         <button type="button" className="button is-info is-light">
+//           Sort alphabetically
+//         </button>
+
+//         <button type="button" className="button is-success is-light">
+//           Sort by length
+//         </button>
+
+//         <button type="button" className="button is-warning is-light">
+//           Reverse
+//         </button>
+
+//         <button type="button" className="button is-danger is-light">
+//           Reset
+//         </button>
+//       </div>
+
+//       <ul>
+//         <ul>
+//           <li data-cy="Good">Dumplings</li>
+//           <li data-cy="Good">Carrot</li>
+//           <li data-cy="Good">Eggs</li>
+//           <li data-cy="Good">Ice cream</li>
+//           <li data-cy="Good">Apple</li>
+//           <li data-cy="Good">...</li>
+//         </ul>
+//       </ul>
+//     </div>
+//   );
+// };
+
+const SORT_FIELD_ALPHABET = 'alph';
+const SORT_FIELD_LENGTH = 'length';
```

Comment:
The task requires using `enum SortType` for sorting options. Currently, string literals are used instead. Consider defining an enum for sort types and using it here.